client/src/connection/itc/script.ts

```diff
@@ -244,17 +244,21 @@ class SASRunner{
     Write-Host "${LineCodes.ResultsFetchedCode}"
   }
 
-  [void]GetDatasetRecords([string]$tableName, [int]$start = 0, [int]$limit = 100) {
+  [void]GetDatasetRecords([string]$tableName, [int]$start = 0, [int]$limit = 100, [string]$sortCriteria = "") {
     $objRecordSet = New-Object -comobject ADODB.Recordset
     $objRecordSet.ActiveConnection = $this.dataConnection # This is needed to set the properties for sas formats.
     $objRecordSet.Properties.Item("SAS Formats").Value = "_ALL_"
 
+    $query = "SELECT * FROM $tableName"
```

Comment:
For the script changes, would love if we could put some quoting/escaping around $tableName and $sortCriteria?